guides/mobile_app_guidance.md

```diff
@@ -0,0 +1,46 @@
+Mobile Application Guidance
+===========================
```

Comment:
Suggest using markdown headers rather than underlining to be consistent with rest of repo

Reply:
OK